plugin-modernizer-core/src/main/resources/META-INF/rewrite/recipes.yml

```diff
@@ -106,6 +106,14 @@ recipeList:
   - io.jenkins.tools.pluginmodernizer.RemoveExtraMavenProperties
 ---
 type: specs.openrewrite.org/v1beta/recipe
+name: io.jenkins.tools.pluginmodernizer.MigrateToJUnit5
+displayName: Migrate to JUnit 5
+description: Migrate tests to JUnit5.
+tags: ['migration', 'testing']
+recipeList:
+  - io.jenkins.tools.pluginmodernizer.core.recipes.AnnotateWithJenkins
```

Comment:
Maybe reference also recipes from : https://github.com/jenkins-infra/plugin-modernizer-tool/issues/857#issuecomment-2714391291 ?

Reply:
I already have a branch with that, but it would make more sense to start from this one.
Thanks.